auth-web/src/components/auth/account-settings/statement/Statements.vue

```diff
@@ -99,6 +99,14 @@
             <span>{{ paymentMethodsDisplay(item.paymentMethods) }}</span>
           </div>
         </template>
+        <template
+          v-if="enableEFTPaymentMethod && hasEFTPaymentMethod"
```

Comment:
I think this change is supposed to be applied to statements in general, not just EFT. 
Maybe confirm this.

Reply:
yup, should be in general